apps/dokploy/server/api/routers/settings.ts

```diff
@@ -657,6 +658,57 @@ export const settingsRouter = createTRPCRouter({
 		}
 		return { status: "not_cloud" };
 	}),
+	setupGPU: adminProcedure
+		.input(
+			z.object({
+				serverId: z.string().optional(),
+			}),
+		)
+		.mutation(async ({ input }) => {
+			if (IS_CLOUD) {
+				throw new Error("GPU setup is not available in cloud mode");
+			}
```

Comment:
I think we can enable this for cloud version, just add a validation here as 

```suggestion
			if (IS_CLOUD && !input.serverId) {
				throw new Error("Select a server to enable the GPU Setup");
			}
```